ouroboros-consensus-test/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -63,6 +66,18 @@ tests = testGroup "Mempool"
       , testProperty "splitAfterTxSize"                    prop_TxSeq_splitAfterTxSize
       , testProperty "splitAfterTxSizeSpec"                prop_TxSeq_splitAfterTxSizeSpec
       ]
+  , testGroup "MempoolPure"
+      [ testProperty "removed transactions are removed"    prop_pure_removeTxs
+      , testProperty "not removed transactions remain"     prop_pure_removeTxs_contra
+      , testProperty "valid added txs == getTxs"           prop_pure_addTxs_getTxs
+      , testProperty "addTxs txs == mapM addTxs txs"       prop_pure_addTxs_one_vs_multiple
+      , testProperty "snapshotTxs == snapshotTxsAfter zeroIdx"
+                                                           prop_pure_snapshotTxs_snapshotTxsAfter
+      , testProperty "result of addTxs"                    prop_pure_addTxs_result
+      , testProperty "Invalid transactions are never added" prop_pure_InvalidTxsNeverAdded
```

Comment:
I know you just copied the existing line below, but that the capitalisation of that line was already inconsistent, so let's fix it now :slightly_smiling_face: 
```suggestion
      , testProperty "invalid transactions are never added" prop_pure_invalidTxsNeverAdded
```